### PR TITLE
openstack:Add log for delete unsuccessful action

### PR DESCRIPTION
**Pull request by jichenjc**
some action unsuccessful but no log at all, this is not helpful
for debug, we need log those information.

### CODE REVIEW

File: pkg/destroy/openstack/openstack_deprovision.go [hunk 1]
```diff
@@ -261,6 +261,7 @@ func deletePorts(opts *clientconfig.ClientOpts, filter Filter, logger logrus.Fie
 		err = ports.Delete(conn, port.ID).ExtractErr()
 		if err != nil {
 			// This can fail when port is still in use so return/retry
+			logger.Debugf("Deleting Port failed: %v", err)
```
Comment (mandre) [inline pkg/destroy/openstack/openstack_deprovision.go]: The trailing spaces cause gofmt job to fail.

File: pkg/destroy/openstack/openstack_deprovision.go [hunk 2]
```diff
@@ -463,6 +467,7 @@ func deleteNetworks(opts *clientconfig.ClientOpts, filter Filter, logger logrus.
 		err = networks.Delete(conn, network.ID).ExtractErr()
 		if err != nil {
 			// This can fail when network is still in use
+			logger.Debugf("Deleting Network failed: %v", err)
```
Comment (mandre) [inline pkg/destroy/openstack/openstack_deprovision.go]: Ditto

Comment (jichenjc) [reply pkg/destroy/openstack/openstack_deprovision.go]: done

**Pull request merged**